### PR TITLE
feat: added gateway and asm ingress k8s resources

### PR DESCRIPTION
**Pull request by yliaog**

### CODE REVIEW

File: 6-appsource/cymbal-bank/frontend/ingress.yaml [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+---
+kind: ServiceExport
+apiVersion: net.gke.io/v1
+metadata:
+  name: asm-ingressgateway
+  namespace: asm-ingress
+
+---
+apiVersion: networking.istio.io/v1alpha3
```
Comment (apeabody) [inline 6-appsource/cymbal-bank/frontend/ingress.yaml:79]:
```suggestion
apiVersion: networking.istio.io/v1beta1
```

Comment (yliaog) [reply 6-appsource/cymbal-bank/frontend/ingress.yaml]: fixed. thanks.

**Review by gtsorbo — APPROVED** [review]

**Pull request merged**